Cargo.toml

```diff
@@ -68,7 +68,6 @@ reth-rpc-types = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.
 ] }
 reth-rpc-types-compat = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.1", default-features = false }
 reth-rpc-eth-types = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.1", default-features = false }
-reth-rpc = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.1", default-features = false }
```

Comment:
why?

Reply:
just not used, lint told me to remove it, don't know why we didn't remove it before

Reply:
exactly yes